project/MiMaFilters.scala

```diff
@@ -5,20 +5,11 @@ object MiMaFilters {
   val LibraryBackwards: Map[String, Seq[ProblemFilter]] = Map(
     // In general we should never have backwards incompatible changes in the library.
     // Only exceptional cases should be added here.
-    Build.previousDottyVersion -> Seq(
-      // This language feature was in 3.4.0-RC1 but will be removed in 3.4.0-RC2
-      ProblemFilters.exclude[MissingFieldProblem]("scala.runtime.stdLibPatches.language#deprecated.ascriptionVarargsUnpacking"),
-      ProblemFilters.exclude[MissingClassProblem]("scala.runtime.stdLibPatches.language$deprecated$ascriptionVarargsUnpacking$"),
-    )
+    Build.previousDottyVersion -> Seq.empty
   )
   val LibraryForward: Map[String, Seq[ProblemFilter]] = Map(
     // Additions that require a new minor version of the library
     Build.previousDottyVersion -> Seq(
-      // This language feature is not in 3.4.0-RC1 but will be added in 3.4.0-RC2
```

Comment:
There is something broken in `3.4.0-RC2`. The changes of #19316 where modified when backporting 
https://github.com/lampepfl/dotty/pull/19316/files#diff-54406bedea7a8bfcd15a49467ca97594ae3c8ef3de104b0ebe5828d146b8ecd4R36 
to
https://github.com/lampepfl/dotty/commit/5fe047b8e825acdcf1392d7b06e4df0cf6f2b80e#diff-54406bedea7a8bfcd15a49467ca97594ae3c8ef3de104b0ebe5828d146b8ecd4R36
This implies that we did not add the expected `3.5` language tag.

Reply:
We have two ways to fix this

1. We fix this and release a `3.4.0-RC3`
2. We do not fix it and revert temporarily (for 3.4.0) the `3.5` version from the main branch.  Then we add it back when we decide to go to `3.5`.